server/settings/imageio.py

```diff
@@ -47,6 +47,11 @@ class NodesModel(BaseSettingsModel):
         title="Nuke Node Class",
         enum_resolver=nuke_node_class_enum
     )
+    custom_class: str = SettingsField(
+        default_factory="",
+        title="Custom Node Class",
+        description="Custom node class not listed above (optional)"
+    )
```

Comment:
This may be nicer by adding a "Custom Class" entry to the `nuke_node_class` enum, and when selected make the `custom_class` attribute visible in settings.

I believe that's done by adding `conditionalEnum=True` to the `nuke_node_class` and adding a `custom_class` value. Then, only when that one is selected the `custom_class` attribute becomes visible in settings.